modules/utils/.gitignore

```diff
@@ -1,3 +1,4 @@
 onnx_driver/
 target/
-output/
\ No newline at end of file
+output/
+downloaded_onnx_package/
```

Comment:
Tbh we could probably move the package to `output/downloaded_onnx` instead. Because it is a byproduct of building, even if its required for the build.

Reply:
I've now removed the reliance on downloads and it's all now in the`target` directory